gcsio/src/main/java/com/google/cloud/hadoop/gcsio/ZeroCopyMessageMarshaller.java

```diff
@@ -135,4 +135,11 @@ private T parseFrom(CodedInputStream stream) throws InvalidProtocolBufferExcepti
   public InputStream popStream(T message) {
     return unclosedStreams.remove(message);
   }
+
+  // Application needs to call stream.close() function to all streams returned.
```

Comment:
nit: Can we make it javadoc ? Also potentially adding javadoc to all public methods

Reply:
Done